docs/README.md

```diff
@@ -21,7 +21,65 @@ more about Community, Certified and Marketplace operators and contribution.
 
 We would love to see your Operator added to this collection. We currently use automated vetting via continuous integration plus manual review to curate a list of high-quality, well-documented Operators. If you are new to Kubernetes Operators start [here](https://sdk.operatorframework.io/build/).
 
-If you have an existing Operator read our contribution guidelines on how to [open a PR](contributing-via-pr.md). Then the community operator pipeline will be triggered to test your Operator and merge a Pull Request.
+If you have an existing Operator read our contribution guidelines on how to [open a PR](users/contributing-via-pr.md). Then the community operator pipeline will be triggered to test your Operator and merge a Pull Request.
+
+## Remove Your Operator
+
+**Before You Begin**
+Ensure your operator follows the [FBC (File-Based Catalog)](users/fbc_onboarding.md#file-based-catalog-onboarding) workflow.  
+Setting fbc.enabled: true in the ci.yaml file is not enough. The operator must be fully onboarded to FBC.  
+For non-FBC (bundle-based) operators, refer to the [FBC onboarding guide](users/fbc_onboarding.md#convert-existing-operator-to-fbc) before continuing.
+
+Depending on your use case, you may:
+- [Remove the operator entirely from all catalogs.](#remove-the-entire-operator-from-the-catalog)
+- [Remove it from specific catalog version(s).](#remove-the-operator-from-specific-catalog-versions)
+- [Remove a single operator version from the catalog.](#remove-a-single-operator-bundle-version)
+
+### Remove the Entire Operator from the Catalog
+
+To remove the operator completely from the catalog:
+
+- Delete the your operator directory from the `operators/` folder.
+- Remove all catalog files related to your operator from the `catalogs/` directory.
+- Open a **single** pull request that includes these changes. Follow our contribution guidelines on how to [open a PR](users/contributing-via-pr.md).
+
+For reference, here’s an [example PR](https://github.com/redhat-openshift-ecosystem/community-operators-prod/pull/5955/files) demonstrating these steps.
+
+### Remove the Operator from Specific Catalog Version(s)
+
+To remove your operator from selected catalog versions:
+
+- In the `catalogs/` directory, delete the your operator related catalog file from the targeted catalog version(s).
+- In `operators/<operator-name>/ci.yaml`, locate the `fbc.catalog_mapping` section and remove the targeted catalog version(s) from the `catalog_names` list.
```

Comment:
This should be done as a first step and then user should run `make catalogs` that hanles a removal of the subdirectories within the `/catalogs`